CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ cmake_minimum_required( VERSION 3.0 )
 # header. And don't set LANGUAGES as we are multi-language and header
 # only, so it's irrelevant.
 project( lyra )
+set(PROJECT_VERSION "1.5.1")
+set(PROJECT_DESCRIPTION "A simple to use, composing, header only, command line arguments parser for C++ 11 and beyond.")
+set(PROJECT_HOMEPAGE_URL "https://github.com/bfgroup/Lyra")
+
+include(GNUInstallDirs)
+string(REPLACE "/${CMAKE_LIBRARY_ARCHITECTURE}" "" CMAKE_INSTALL_LIBDIR_ARCHIND "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
Why are we using `CMAKE_LIBRARY_ARCHITECTURE` here?

Reply:
It is the impl detail of `GNUInstallDirs`. `CMAKE_INSTALL_LIBDIR` is composed of it and other stuff, usually `lib`. I have tried to set it to `""` before including `GNUInstallDirs` without any success. IMHO instead `GNUInstallDirs` should also create arch-independent vars.